telemetry.go

```diff
@@ -24,12 +22,12 @@ type telemetryAgent interface {
 }
 
 // No-op agent in case the proper variables are not set
-type noopTelemetryAgent struct{}
+type nopTelemetryAgent struct{}
```

Comment:
Was the rename on purpose?

Reply:
Yes, I saw `nop` used instead of `noop` in another project and it just looked a bit cleaner so I changed it globally